.gitignore

```diff
@@ -0,0 +1,3 @@
+# .gitignore
+node_modules/
+test.md
```

Comment:
- [ ] Great job adding test.md to gitignore file. Kindly remove test.md from your repo since it is meant to be ignored by git. If a file is ignored by git, it won't be added to your repo.